mmv1/products/apigee/terraform.yaml

```diff
@@ -47,6 +47,16 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       min_version: beta
       # Resource creation race
       skip_vcr: true
+    - !ruby/object:Provider::Terraform::Examples
+      name: "apigee_organization_retention_test"
+      primary_resource_id: "org"
+      test_env_vars:
+        org_id: :ORG_ID
+        billing_account: :BILLING_ACCT
+      skip_docs: true
+      min_version: beta
```

Comment:
Is the feature beta-only? That may cause some issues if the parameter needs to exist in the URL string, as that is shared across beta and GA

Reply:
No, right now Apigee has no beta API, but running the full Terraform acceptance tests for this feature depends on a beta-only resource.